WoWPro_Leveling/Wrath/Horde/WOTLK_Borean_Tundra.lua

```diff
@@ -251,26 +243,23 @@ T Defeat the Gearmaster |QID|11909|M|77.3,36.9|N|To Greatmother Taiga.|
 A Blending In |QID|11633|PRE|11630|M|75.9,37.2|N|From Chieftain Wintergale.|
-;QO
-N Scout Spire of Pain |QID|11633|QO|3|U|34782|N|Head to the city and use the cloak. You're invisible to everything except abominations. If they get in the way or detect you, kill them and re-equip the cloak. Go east to the Spire of Pain. You can be mounted while using the cloak.|M|89.2,28.7|
-N Scout Spire of Blood |QID|11633|QO|2|U|34782|N|Go east to the Spire of Blood - you need to climb the outside stairs to either side of the normal doorway. |M|88,21|
+C Scout Spire of Pain |QID|11633|QO|3|U|34782|N|Head to the city and use the cloak. You're invisible to everything except abominations. If they get in the way or detect you, kill them and re-equip the cloak. Go east to the Spire of Pain. You can be mounted while using the cloak.|M|89.2,28.7|
+C Scout Spire of Blood |QID|11633|QO|2|U|34782|N|Go east to the Spire of Blood - you need to climb the outside stairs to either side of the normal doorway. |M|88,21|
 C Blending In |QID|11633|QO|1|U|34782|N|Go east to the Spire of Decay. |M|84,22|
-;N
 T Blending In |QID|11633|M|75.9,37.2|N|To Chieftain Wintergale. Don't forget to equip your regular cloak again.|
 A Words of Power |QID|11640|PRE|11633|M|75.9,37.2|N|From Chieftain Wintergale.|
 A Neutralizing the Cauldrons |QID|11647|M|76.0,37.3|N|From Sage Aeire.|
 A A Courageous Strike |QID|11641|M|76.0,37.3|N|From Durm Icehide.|
 C A Courageous Strike |QID|11641|N|Kill En'kilah Ghouls and Necromancers.|S|M|85.76,26.79|
-;QO
 K High Priest Talet-Kha |QID|11640|QO|2|N|Head into En'kilah again. Go east to the Spire of Pain, kill the Mystical Webbings and the priest will attack you. Loot the scroll off his body.|M|89,29|
-N Neutralize the East Cauldron |QID|11647|QO|1|U|34806|N|It's just to the south as you exit the spire. Walk up to it and use the totem.|M|87.68,29.96|
+C Neutralize the East Cauldron |QID|11647|QO|1|U|34806|N|It's just to the south as you exit the spire. Walk up to it and use the totem.|M|87.68,29.96|
 K Darkfallen Bloodbearer |QID|11654|L|34815|N|Kill and loot Darkfallen Bloodbearer. He patrols between the Spire of Blood and the entrance.|M|88,22|T|Darkfallen Bloodbearer|
 A The Spire of Blood |QID|11654|U|34815|M|87.7,22.0|N|Right click the Vial of Fresh Blood to accept the quest.|
 T The Spire of Blood |QID|11654|N|To Snow Tracker Grumm. He's on the lower floor of the Spire of Blood.|M|87.6,20.0|
 A Shatter the Orbs! |QID|11659|PRE|11654|M|87.6,20.0|N|From Snow Tracker Grumm.|
 C Shatter the Orbs! |QID|11659|N|There's a bunch on the first floor and one on the second.|
 K High Priest Andorath |QID|11640|QO|3|N|He's on the upper floor, loot the scroll off his body.|M|88.09,20.91|
-N Neutralize the Central Cauldron |QID|11647|QO|2|U|34806|N|Head a bit SW when you exit the Spire of Blood.|M|86,23|
+C Neutralize the Central Cauldron |QID|11647|QO|2|U|34806|N|Head a bit SW when you exit the Spire of Blood.|M|86,23|
```

Comment:
Coordinates without decimal places are awful.  Even if you just peruse wowhead and find coordinates with one decimal place or a comment with two, it is is **waaaay** better.  If you really mean 86, then use 86.00 to mark it as being deliberate.